codingTest_kjy/나머지1이되는수찾기.js

```diff
@@ -0,0 +1,9 @@
+function solution(n) {
+    var answer = 0;
+   for(let x =0; x < n; x++) {
+       if(n % x === 1) {
+           return x;
```

Comment:
```suggestion
   for(let i = 0; i < n; i++) {
       if(n % i === 1) {
           return i;
```
크게 상관은 없는데 보통 for문 내에서 국룰처럼 선언되는 변수의 이름은 i입니다. for문이 중첩 될 수록 i, j, k, l, m, n 의 순서로 사용하시면 될 것 같아요.